graphql/resolvers.go

```diff
@@ -3101,6 +3101,21 @@ func (r *versionResolver) ChildVersions(ctx context.Context, v *restModel.APIVer
 		for _, cp := range childPatchIds {
 			v, err := r.Query().Version(ctx, cp)
 			if err != nil {
+				//before erroring due to the version being nil or not found,
+				// fetch the child patch to see if it's activated
+				p, err := patch.FindOneId(cp)
+				if err != nil {
+					return nil, err
+				}
+				if p == nil {
+					return nil, gimlet.ErrorResponse{
+						StatusCode: http.StatusNotFound,
+						Message:    fmt.Sprintf("patch with id %s not found", cp),
+					}
+				}
+				if p.Version == "" {
```

Comment:
This runs in a for loop. Why do we return nil here? If only one child patch is unactivated this would return nil for all of them.

Reply:
shoots! you're right, thanks for spotting that!

Reply:
done